tests/e2e/run-scanner-v4.bats

```diff
@@ -70,6 +74,258 @@ teardown() {
     run remove_existing_stackrox_resources
 }
 
+# We are using our own deploy function, because we want to have the flexibility to patch down resources
+# after deployment. Without this we are only able to special-case local deployments and CI deployments,
+# but not, for example, manual testing on Infra.
+#
+# shellcheck disable=SC2120
+_deploy_stackrox() {
```

Comment:
The risk here is that these versions of the functions (`_deploy_stackrox`, `_deploy_central`, `_deploy_sensor`), which are currently almost identical copies of `deploy_stackrox` etc, will in time start to diverge.

Which leads to the question, why can't we just use the original ones and then call the `patch_down_*` functions? I suppose it has something to do with the comment above about testing on Infra, but I'm not sure that I fully understand.

Reply:
`deploy_stackrox` does several things: `deploy_central` and `deploy_sensor` plus waiting for readiness of these services. It is `deploy_stackrox` which failed to succeed because of the huge resource requirements. So, I cannot simply patch down resources after `deploy_stackrox`, because that one fails already. I need to patch down right after `deploy_central` and `deploy_sensor`.

I agree with the diverging problem. But I figured after all the work that I was already able to merge with the help of Automation team, I now need to focus on the actual test and get it out of the door to unblock several other tickets.

Would it be ok to create a backlog ticket for extending `deploy_stackrox` so that it can configure resource requirements more flexible? I see several potential ways of doing this.

Reply:
Ah I understand now. Yes it's OK if you create a follow-up ticket.